static/app/components/checkbox.tsx

```diff
@@ -23,12 +23,16 @@ interface Props extends Omit<CheckboxProps, 'checked' | 'size'> {
   size?: FormSize;
 }
 
-type CheckboxConfig = {borderRadius: string; box: string; icon: string};
+type CheckboxConfig = {
+  borderRadius: string;
+  box: string;
+  icon: IconSize;
+};
 
 const checkboxSizeMap: Record<FormSize, CheckboxConfig> = {
-  xs: {box: '12px', icon: '10px', borderRadius: '2px'},
-  sm: {box: '16px', icon: '12px', borderRadius: '4px'},
-  md: {box: '22px', icon: '16px', borderRadius: '6px'},
+  xs: {box: '12px', borderRadius: '2px', icon: 'xs'},
```

Comment:
I had to change this. This will be a problem cc @malwilley.

We should probably just use custom SVGs for this or something

Reply:
We can define the sizes as pixel values and underneath pass them as the `legacySize` prop instead of `size`
https://github.com/getsentry/sentry/blob/evanpurkhiser/ref-ui-enforce-iconsize-enum-usage/static/app/components/checkbox.tsx#L64

Reply:
yeah.. urgh